cityofaustin/joplin
# 4076 localize docs

# Description

Content needs to be able to add spanish documents along with English documents. 

This PR adds the field and also adjusts the schema so a custom resolver will return the correct document. 

Something I learned working on this: I added the fields, added some documents. Then changed the fields (adding blank is true to the spanish field) and then could not get the resolvers to work with the exact code that Nick had working on his machine. I finally realized that I needed to run migrations in order for the schema to work. 

Thanks to Eric and Nick for helping me with this PR. 







# Types of changes



# How can this be tested?



 

https://janis-4076-local-docs.netlify.com/es/health-safety/policing-in-austin/police-feedback-and-records/official-complaint-and-discipline-documents/#1
is the janis

and https://joplin-pr-4076-local-docs.herokuapp.com/admin/pages/search/ is the linked joplin.

I replaced the first entry on janis (the last one on joplin) with a fake spanish pdf, you can see it by clicking the first document on janis. other documents that were not replaced with a spanish one still have the english document.

# Checklist:
